packages/component-library/stories/InlineNotification.stories.tsx

```diff
@@ -136,6 +147,19 @@ export const DismissibleMultiline = () => (
 
 DismissibleMultiline.storyName = "Dismissible, Multiline"
 
+export const DismissibleForceMultiline = () => (
+  <InlineNotification
+    type="negative"
+    title="Negative"
+    automationId="notification1"
+    forceMultiline
+  >
+    {forceMultilineText}
+  </InlineNotification>
+)
+
+DismissibleMultiline.storyName = "Dismissible, Multiline"
```

Comment:
This line isn't doing anything currently. It needs to be updated with the correct story var and I think it's worth being explicit about calling this '_Forced_ Multiline'